Web/Pages/CCGMember/Compare/Delta.cshtml

```diff
@@ -0,0 +1,303 @@
+                else if (match3.Count > match1.Count)
+                {
+                    @for (int i = 0; i < match3.Count; i++)
+                    {
+                        if (i < match1.Count)
+                        {
+                            <tr>
+                                <td width="50%">@(isEnglish?match1[i].CompetencyNameEng:match1[i].CompetencyNameFre)</td>
+                                <td width="50%">@(isEnglish?match3[i].CompetencyNameEng:match3[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                        else
+                        {
+                            <tr>
+                                <td width="50%">@null</td>
+                                <td width="50%">@(isEnglish?match3[i].CompetencyNameEng:match3[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                    }
+                    <tr>
+                        <td width="50%"></td>
+                        <td width="50%"></td>
+                    </tr>
+                }
+                else
+                {
+                    <tr>
+                        <td width="50%"></td>
+                        <td width="50%"></td>
+                    </tr>
+                }
+            </table>
+            <h3>@Localizer["DIFFERING COMPETENCIES"]</h3>
+            <table class="table table-hover table-bordered">
+                <tr>
+                    <th style="background-color:#f2f2f2" width="70%">@Localizer["CERTIFICATIONS"]</th>
+                    <th class="text-center bg-success">@Localizer["Current Position"]</th>
+                    <th class="text-center bg-info">@Localizer["Selected Position"]</th>
+                </tr>
+                <tbody>
+                    @if (diffcert.Count == 0)
+                    {
+                        <tr>
+                            <td width="70%"></td>
+                            <td class="text-center bg-success"></td>
+                            <td class="text-center bg-info"></td>
+                        </tr>
+                        <tr>
+                            <td width="70%"></td>
+                            <td></td>
+                            <td></td>
+                        </tr>
+                    }
+                    else
+                    {
+                        @foreach (var d in diffcert)
+                        {
+                            <tr>
+
+                                <td width="70%">@d.NameEng</td>
+                                <td class="text-center bg-success"></td>
+                                <td class="text-center bg-info"></td>
+                            </tr>
+                        }
+                        <tr>
+                            <td width="70%"></td>
+                            <td></td>
+                            <td></td>
+                        </tr>
+                    }
```

Comment:
Example of simplifying markup
- [ ] The empty row in both the if/else can be moved outside the statements since it's the same in both
- [ ] The else statement isn't required since it won't print any rows when diffcert is empty.